src/west/manifest.py

```diff
@@ -2432,6 +2432,10 @@ def _load_project(self, pd: Dict, url_bases: Dict[str, str],
                             f'normalizes to {ret_norm}, which escapes '
                             f'the workspace topdir')
 
+        if os.path.commonpath([ret_norm, util.WEST_DIR]) != '':
```

Comment:
`commonpath` and the negation feel too convoluted to me. I  think this is simpler hence more readable (TESTED)

```suggestion
        if Path(ret_norm).parts[0] == util.WEST_DIR:
```

Also: this is a small step towards the `pathlib` future were paths are objects and not just strings any more :-)

Reply:
That look much cleaner.